catalogd/cmd/catalogd/main.go

```diff
@@ -263,6 +263,10 @@ func main() {
 	}
 
 	unpackCacheBasePath := filepath.Join(cacheDir, source.UnpackCacheDir)
+	if err := os.RemoveAll(unpackCacheBasePath); err != nil {
```

Comment:
would it make sense to nuke `cacheDir` (above) instead? And, should we also make a similar change to o-c for the bundle cache?

Reply:
Yes, but the cache dir itself is a volumeMount. So we likely need to nuke all of the contents of the cache dir without actually nuking the cache dit itself.